src/api/app/controllers/webui/users/rss_token_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Webui
+  module Users
+    class RssTokenController < WebuiController
```

Comment:
Same reason as @evanrolfe hold when we created **users/bs_requests_controller**, **projects/bs_requests_controller**, **projects/public_key_controller** and **projects/ssl_certificate_controller**... for having CRUD controllers.